Makefile

```diff
@@ -3,6 +3,8 @@ SYNO_ARCH ?= "amd64"
 SYNO_DSM ?= "7"
 TAGS ?= "latest"
 
+PLATFORM ?= "flyio" ## flyio==linux/amd64. Set to "" to build all platforms.
```

Comment:
I think this is the only supported platform, might want to add support for arbitrary platforms to mkctr at some point.

Reply:
Makes sense 👍🏼 I added the comment in case anyone needs to build arm, they would at least know how to build for all.